app/assets/javascripts/spotlight/reindex_monitor.js

```diff
@@ -48,6 +48,10 @@ Spotlight.onLoad(function() {
         .attr('aria-valuenow', percentage)
         .css('width', percentage + '%')
         .text(percentage + '%');
+
+      if (data.completed) {
```

Comment:
Right you are (maybe suggesting that field is poorly named..). I'll just push our `finished` attribute through the JSON and use that.